### PR TITLE
Add inspect command

### PR DESCRIPTION
**Pull request by Konboi**
Add `launchable inspect subset` command to check subset result.

e.g) launchable/cli 
```
$ launchable inspect subset --subset-id <subset-id>
|   Order | Test Path                                   | In Subset   |   Estimated duration (sec) |
|---------|---------------------------------------------|-------------|----------------------------|
|       1 | file=tests/test_plugin.py                   | ✔           |                      0.001 |
|       2 | file=tests/test_runners/test_bazel.py       | ✔           |                      0.001 |
|       3 | file=tests/test_runners/test_gradle.py      | ✔           |                      0.001 |
|       4 | file=tests/test_version.py                  | ✔           |                      0.001 |
|       5 | file=tests/utils/test_click.py              | ✔           |                      0.001 |
|       6 | file=tests/utils/test_logger.py             | ✔           |                      0.001 |
|       7 | file=tests/test_runners/test_rspec.py       | ✔           |                      0.001 |
|       8 | file=tests/test_session.py                  | ✔           |                      0.001 |
|       9 | file=tests/commands/test_verify.py          | ✔           |                      0.001 |
|      10 | file=tests/commands/record/test_tests.py    | ✔           |                      0.001 |
|      11 | file=tests/test_runners/test_robot.py       | ✔           |                      0.001 |
|      12 | file=tests/test_runners/test_minitest.py    | ✔           |                      0.001 |
|      13 | file=tests/test_runners/test_behave.py      | ✔           |                      0.001 |
|      14 | file=tests/commands/record/test_commit.py   | ✔           |                      0.001 |
|      15 | file=tests/test_runners/test_go_test.py     | ✔           |                      0.001 |
|      16 | file=tests/test_runners/test_ant.py         | ✔           |                      0.001 |
|      17 | file=tests/commands/record/test_session.py  |             |                      0.001 |
|      18 | file=tests/utils/test_authentication.py     |             |                      0.001 |
|      19 | file=tests/utils/test_http_client.py        |             |                      0.001 |
|      20 | file=tests/data/pytest/tests/test_funcs1.py |             |                      0.001 |
|      21 | file=tests/test_runners/test_ctest.py       |             |                      0.001 |
|      22 | file=tests/test_runners/test_maven.py       |             |                      0.001 |
|      23 | file=tests/test_runners/test_cypress.py     |             |                      0.001 |
|      24 | file=tests/utils/test_gzipgen.py            |             |                      0.001 |
|      25 | file=tests/utils/test_file_name_pattern.py  |             |                      0.001 |
|      26 | file=tests/test_runners/test_googletest.py  |             |                      0.001 |
|      27 | file=tests/data/pytest/tests/test_funcs2.py |             |                      0.001 |
|      28 | file=tests/test_runners/test_adb.py         |             |                      0.001 |
|      29 | file=tests/commands/test_helper.py          |             |                      0.001 |
|      30 | file=tests/test_testpath.py                 |             |                      0.001 |
|      31 | file=tests/test_runners/test_pytest.py      |             |                      0.001 |
|      32 | file=tests/test_runners/test_nunit.py       |             |                      0.001 |
```

### CODE REVIEW

**Comment by Konboi** [comment]
I need to confirm two things.

1) How about change the eliminated duration unit from min to sec?
Because I think almost tests durations are under 1 minute.

2) What do you think this test path format?
We don't require the test framework name when run this command now.  We can't know the test path format like the subset command. So I just join response with '#'.

**Comment by awilkes** [comment]
> @awilkes I want to confirm two things.
> 
>     1. How about changing the eliminated duration unit from min to sec?
>        Because I think test duration that per test is usually not so long. so sec is easier to understand than min.

Good idea! 👍 

>     2. What do you think about this test path format?
>        We don't require the test framework name when running this command now.  In other words, we can't know the test path format like the subset command. So I just join the response with '#' e.g) ` file=tests/test_runners/test_nunit.py` or `package=src/test/java/com/ninjinkun#target=mylib_test7` . Should we require to set the test framework name for the users?

I think joining with `#` is fine 👍

File: launchable/commands/inspect/subset.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    click.echo(tabulate(rows, header, tablefmt="github"))
+
+
+def convert_row(list: List[Dict], order: int, is_subset: bool):
+    """
+    list: testPaths or rest in response to a get subset API
+    order: start number of order
+    is_subset: in subset or not
+    """
+    rows = []
```
Comment (ninjinkun) [inline launchable/commands/inspect/subset.py:49]: You can use `enumerate()` to find indices of the list.
```suggestion
return [[order + i, "#".join([path["type"] + "=" + path["name"]
                                      for path in l["testPath"]]), "✔" 
                                      if is_subset else "", "{:0.4f}".format(l["duration"] / 1000)] 
                                      for i, l in enumerate(list)]
```

**Review by ninjinkun — APPROVED** [review]
I'll leave it up to you to use my idea.

File: launchable/commands/inspect/subset.py (hunk 1 above)
Comment (Konboi) [reply launchable/commands/inspect/subset.py:49]: TIL: enumerate

Thank you!

**Pull request merged**